finesse/hardware/dummy_em27_scraper.py

```diff
@@ -0,0 +1,28 @@
+"""This module provides an interface to a dummy EM27 monitor."""
+import logging
+from importlib import resources
+
+from .em27_scraper import EM27Error, EM27Scraper
+
+
+class DummyEM27Scraper(EM27Scraper):
+    """An interface for testing monitoring EM27 properties."""
+
+    def __init__(self) -> None:
+        """Create a new EM27 property monitor."""
+        dummy_em27_fp = resources.files("finesse.hardware").joinpath("diag_autom.htm")
```

Comment:
In our course of software engineering [we suggest that loading data in the constructor is a bad practice](https://imperialcollegelondon.github.io/grad_school_software_engineering_course/l1-05-abstractions/index.html) as there's a hidden dependency within the constructor that prevents initialising the class if not present - if should be done in a factory method. I don't have strong feelings - and I don't suggest to change this now as it will require significant updates in several places - but I just point this out for the future.